projects/plugins/crm/tests/codeception/_support/Module/WPBrowserMethods.php

```diff
@@ -7,6 +7,11 @@
 use Symfony\Component\BrowserKit\Cookie;
 use function GuzzleHttp\Psr7\build_query;
 
+/**
+ * Config details
+ *
+ * @property array $config
+ */
```

Comment:
Similar, although this one didn't have an old class doc comment to be overridden.
```suggestion
/**
 * @property array $config Config details.
 */
```

Reply:
ef6578b4ba